### DISCUSSION TITLE
Error : Capabilities "Object literal may only specify known properties, and 'capabilities' does not exist in type "Testrunner" ".

### DISCUSSION THREAD

**Discussion by Dharmishta12** [webdriverio/webdriverio#13485] [Q&A] [answered]
I have setup a new project and installed webdriver v9 getting this error over capabilties : Object literal may only specify known properties, and 'capabilities' does not exist in type "Testrunner". 

This is my written code which was autogenerate file:

export const config: Options.Testrunner ={
     //code..
     capabilities :[{
        browserName: 'chrome',
     }] 
}

**Comment by christian-bromann** [comment, marked as answer]
@Dharmishta12 there is likely a problem in our documentation, moving forward we recommend to type your configuration as following:

```ts
export const config: WebdriverIO.Config = {
  // ...
}
```

Let me know if you have any further questions.

**Comment by Dharmishta12** [comment]
Thankyou so much .It got resolved

**Comment by sperezc06** [comment]
happened to me! 
fixed, thanks

**Comment by Lovishkumarbansal** [comment]
I have a wdio.shared.conf.ts file and code is 

import 'tsconfig-paths/register';
import type { Options } from '@wdio/types';
// exports.config = {
    // export const config: Options.Testrunner = {
        export const **config:** WebdriverIO.Config ={
    runner: 'local',
    tsConfigPath: './tsconfig.json',
    
    exclude: [],
    maxInstances: 1,
    
    logLevel: 'info',
    bail: 0,
    waitforTimeout: 10000,
    connectionRetryTimeout: 120000,
    connectionRetryCount: 3,
    services: [['appium', {
        command: 'appium',
        args: {
            address: 'localhost',
            port: 4724,
            relaxedSecurity: true,
        }
    }]],
    framework: 'cucumber',
    specFileRetries: 1,
    specFileRetriesDelay: 0,
    specFileRetriesDeferred: false,
    reporters: ['spec', ['allure', { outputDir: 'allure-results' }], 'cucumberjs-json'],
    cucumberOpts: {
        require: ['./test/stepDefinitions/**/*.ts'],
        backtrace: false,
        requireModule: [],
        dryRun: false,
        failFast: false,
        name: [],
        snippets: true,
        source: true,
        strict: false,
        tagExpression: '',
        timeout: 60000,
        ignoreUndefinedDefinitions: false
    },

    
};
export { **config** };

Getting error on Highlighted config
